Task_TheGridProblem/index.css

```diff
@@ -1 +1,72 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */
+ * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      body {
+        display: grid;
+        place-items: center;
+      }
+
+      .container {
+        color: #fff;
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 1fr 0.5fr 1fr;
+        gap: 10px;
+        height: 700px;
+        width: 700px;
+        padding: 20px;
+      }
+
+      .item-1 {
+        grid-column: 1 / 2;
+        grid-row: 1 / 2;
+      }
+
+      .item-2 {
+        grid-column: 2 / 3;
+        grid-row: 1 / 2;
+      }
+
+      .item-3 {
+        grid-column: 3 / 3;
+        grid-row: 1 / 2;
+      }
+
+      .item-4 {
+        grid-column: 1/ 3;
+        grid-row: 4/2;
+      }
+
+      .item-5 {
+        grid-column: 3 / 3;
+        grid-row: 2 / 3;
+      }
+
+      .item-6 {
+        grid-column: 3 / 3;
+        grid-row: 4 / 3;
+      }
+
+      .item {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: #f2f2f2;
+        font-size: 30px;
+        font-weight: bold;
+        height: 100%;
+        width: 100%;

```

Comment:
There is a duplicate 'background-color' property on line 62. It should be removed to avoid ambiguity.